src/main/webapp/tslint.json

```diff
@@ -130,6 +130,7 @@
       ]
     },
     "no-conflicting-lifecycle": true,
+    "no-explicit-any": false,
```

Comment:
Not a blocker by any means considering the deadlines, but in general, avoiding `any` is nice

Reply:
I know! I only did it because of the gapi but I do plan to change this in the future